deployments/compose/etc/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
1. This is bash not sh because you use $((..))
2. Rename the file to identify it is entrypoint for HiveThriftServer, because it is under etc folder.

Reply:
Done, Thanks